src/vireshJJun20/ArithOps.java

```diff
@@ -0,0 +1,81 @@
+package vireshJJun20;
+
+class ArithOps{
+
+	double sum(double a, double b, double c){
+		double smAnswer;
+		smAnswer= a+b+c;
+		System.out.println(" - Sum: " + smAnswer);
+		return smAnswer;
+	}
+	
+	double average(double a, double b, double c){
+		ArithOps ar= new ArithOps();
```

Comment:
No need to create new object. non static to non static method can be called directly.

Reply:
Removed the step 13 (object creation with ref variable ar)